external-crates/move/crates/move-package/tests/test_runner.rs

```diff
@@ -114,6 +129,15 @@ impl Test<'_> {
     }
 }
 
+/// Return the dependencies contained in the file at `path`, if any
+fn load_implicits(path: &Path) -> Dependencies {
```

Comment:
nit (feel free to ignore): 
```suggestion
fn load_implicits_from_file(path: &Path) -> Dependencies {
```